src/js/dropdown-menu/dropdown-menu-view.js

```diff
@@ -102,7 +114,22 @@ define([
             },
 
             clickCloseDropdown: function(event, context) {
-                var $el = $(event.target);
+                var $el = $(event.target),
```

Comment:
You probably want to use ``$(event.currentTarget)`` to always get the object that you assigned the listener to:

http://stackoverflow.com/questions/5921413/difference-between-e-target-and-e-currenttarget